jsonrpc/txpool_endpoint.go

```diff
@@ -22,8 +25,8 @@ type TxPool struct {
 }
 
 type ContentResponse struct {
-	Pending map[types.Address]map[uint64]*txpoolTransaction `json:"pending"`
-	Queued  map[types.Address]map[uint64]*txpoolTransaction `json:"queued"`
+	Pending map[types.Address]map[uint64]*transaction `json:"pending"`
+	Queued  map[types.Address]map[uint64]*transaction `json:"queued"`
```

Comment:
Does it mind since transaction struct has a couple of more fields comparing to `txpoolTransation` previously had (
E.g. `V`, `R`, `S`, `Type`, `ChainID`)? Should we add `omitempty` tag maybe?

Reply:
I have test some public geth nodes (like https://rpc-sepolia.rockx.com/). Most of public nodes do not allow txpool_content method, but the ones that do, they return all these fields.
Tested with postman 
```
{
  "jsonrpc": "2.0",
  "method": "txpool_content",
  "params": [
  ],
  "id": 1
}
```